packages/client/modules/teamDashboard/components/AgendaAndTasks/AgendaAndTasks.tsx

```diff
@@ -130,7 +130,11 @@ const AgendaAndTasks = (props: Props) => {
               <StyledLabelHeading>{'Team Agenda'}</StyledLabelHeading>
               <CloseAgenda hideAgenda={hideAgenda} teamId={teamId} />
             </AgendaHeader>
-            <AgendaListAndInput dashSearch={dashSearch || ''} team={team!} />
+            <AgendaListAndInput
+              agendaItems={agendaItems}
```

Comment:
you're already passing down team, and agendaItems is a sub-object of team, so no need to pass them both down